test/release/examples/primers/fact.h

```diff
@@ -0,0 +1,10 @@
+#ifndef FACT_H
+#define FACT_H
+int fact(int x);
+typedef struct _data {
+    int  x;
+} data;
+data* getNewData(void);
+data* fact_d(data* d);
+data* getDataStructPtr(void);
+#endif /* FACT_H */
```

Comment:
This is a little squashed together, it'd be nice to insert some newlines.  Maybe after:
- the `#define`
- the first function definition
- the struct definition
- and the last function definition?